runtime/src/bpf_helper.cpp

```diff
@@ -3,14 +3,22 @@
  * Copyright (c) 2022, eunomia-bpf org
  * All rights reserved.
  */
+#if __APPLE__
+#include <_types/_uint64_t.h>
```

Comment:
What's the purpose of this header?

Reply:
had added it before to use `uint64` but later on made my custom implementation. removing it.